Documentation/api-reference.md

```diff
@@ -0,0 +1,339 @@
+# Adobe Experience Platform Identity for Edge Network Extension - Android
+
+## Prerequisites
+
+Refer our [Getting Started Guide](getting-started.md)
+
+## API reference
+
+| APIs                                           |
+| ---------------------------------------------- |
+| [extensionVersion](#extensionVersion)          |
+| [getExperienceCloudId](#getExperienceCloudId)  |
+| [getIdentities](#getIdentities)                |
+| [getUrlVariables](#getUrlVariables)            |
+| [registerExtension](#registerExtension)        |
+| [removeIdentity](#removeIdentity)              |
+| [resetIdentities](#resetIdentities)            |
+| [updateIdentities](#updateIdentities)          |
+
+------
+
+### extensionVersion
+
+The extensionVersion() API returns the version of the Identity for Edge Network extension.
+
+#### Java
+
+##### Syntax
+```java
+public static String extensionVersion()
+```
+
+##### Example
+```java
+String extensionVersion = Identity.extensionVersion();
+```
+------
+
+### getExperienceCloudId
+
+This API retrieves the Experience Cloud ID (ECID) that was generated when the app was initially launched. This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall.
+
+> The ECID value is returned via the [AdobeCallback](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#public-classes). When [AdobeCallbackWithError](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#public-classes) is provided to this API, the timeout value is 500ms. If the operation times out or an unexpected error occurs, the `fail` method is called with the appropriate [AdobeError](https://aep-sdks.gitbook.io/docs/foundation-extensions/mobile-core/mobile-core-api-reference#public-classes).
```

Comment:
I am thinking we should remove the external links to the MobileCore classes, and use simple code like quotes instead, e.g. `AdobeCallbackWithError`